lib/resources/etc_libuser.rb

```diff
@@ -0,0 +1,108 @@
+require 'utils/simpleconfig'
+require 'utils/object_traversal'
+
+module Inspec::Resources
+  class EtcLibUser < Inspec.resource(1)
+    name 'etc_libuser'
+    desc 'Use the etc_libuser InSpec audit resource to check the configuration of /etc/libuser.conf.
+          Due to limitations of the RSpec engine and for consistency, please use the method described here
```

Comment:
I don't think this needs to be directly in the resource documentation. Perhaps you can suggest a different place for it, but this warning applies to anything that may have a `.` character in it across many resources and I would not want to litter this warning throughout all resource documentation.